dvc/stage/run.py

```diff
@@ -79,9 +79,9 @@ def _run(stage, executable, cmd, checkpoint_func, **kwargs):
         threading.current_thread(),
         threading._MainThread,  # pylint: disable=protected-access
     )
+    old_handler = None
 
     exec_cmd = _make_cmd(executable, cmd)
-    old_handler = None
```

Comment:
@karajan1001 What is this change about?

Reply:
@karajan1001 If you can explain this change, I'm happy to approve. Thanks!

Reply:
Actually not anything affected by this. I will undo this.